port-forward-fun.conf

```diff
@@ -1,19 +0,0 @@
-##################### Notes ########################
-#                                                  #
-#  So, you wanna run a single Node server, huh?    #
-#  Here's what the Apache server might look like   #
-#                                                  #
-#  This will assume an Express server is already   # 
-#                running on Port 3000              #
-#                                                  #
-####################################################
-
-<VirtualHost *:80>
-    ServerAdmin          admin@mycoolsite.com
-    ServerName           mycoolsite.com
-    ProxyPreserveHost    On
-    ProxyPass            "/" "http://127.0.0.1:3000/"
-    ProxyPassReverse     "/" "http://127.0.0.1:3000/"
```

Comment:
We do need both of them however!

The "/" denotes that we want to serve up at the root of our site eg; mycoolsite.com rather than say, nice.mycoolsite.com or mycoolsite.com/fun

The second parameter "http://127.0.0.1:3000/" is only handled internally, since it's communicating with localhost and basically says serves up the port 3000 server as if it were the root of http://mycoolsite.com

http://httpd.apache.org/docs/current/mod/mod_proxy.html#proxypassreverse